lib/dotcom_web/live/system_status.ex

```diff
@@ -18,23 +18,26 @@ defmodule DotcomWeb.Live.SystemStatus do
   @date_time_module Application.compile_env!(:dotcom, :date_time_module)
 
   def render(assigns) do
+    now = @date_time_module.now()
+
     live_alerts =
       subway_route_ids()
-      |> Alerts.Repo.by_route_ids(@date_time_module.now())
-      |> Enum.filter(&SystemStatus.status_alert?(&1, @date_time_module.now()))
+      |> Alerts.Repo.by_route_ids(now)
+      |> Enum.filter(&SystemStatus.status_alert?(&1, now))
 
     assigns =
       assigns
       |> assign(:alerts, live_alerts)
       |> assign(:examples, alerts_examples())
+      |> assign(:now, now)
 
     ~H"""
     <h1>Live Data</h1>
-    <.example_table alerts={@alerts} />
+    <.example_table alerts={@alerts} now={@now} />
 
     <h1>Examples</h1>
     <div :for={example <- @examples} class="mb-4">
-      <.example_table alerts={example.alerts} />
+      <.example_table alerts={example.alerts} now={@now} />
```

Comment:
Minor: the way it was set up before guaranteed that the 8:30PM scheduled alert would always show up as `8:30 PM`. As this code is, if we look at this after 8:30 on a given day, then it'll render as current.

Minor because if we're looking at the preview page after 8:30 PM, maybe that's a sign that it's time to sign off for the day 😅